app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/date_input.scala.html

```diff
@@ -0,0 +1,77 @@
+@*
+* Copyright 2016 HM Revenue & Customs
```

Comment:
Is it not possible use / adapt the existing input_date view helper? This looks a lot more complicated. If not, the input_date helper is no longer in use so should probably be deleted.

Also, is there a reason for splitting the date inputs across date_input and date_fields?